qbft/spectest/tests/messages/rc_invalid_prepared_value.go

```diff
@@ -9,13 +9,9 @@ import (
 
 // RoundChangeDataInvalidPreparedValue tests PreparedRound != NoRound && PreparedValue == nil
 func RoundChangeDataInvalidPreparedValue() *tests.MsgSpecTest {
-	msg := testingutils.SignQBFTMsg(testingutils.Testing4SharesSet().Shares[1], types.OperatorID(1), &qbft.Message{
-		MsgType:    qbft.RoundChangeMsgType,
-		Height:     qbft.FirstHeight,
-		Round:      10,
-		Identifier: []byte{1, 2, 3, 4},
-		Data:       testingutils.RoundChangePreparedDataBytes(nil, 2, nil),
-	})
+	ks := testingutils.Testing4SharesSet()
```

Comment:
it doesn't test what it should

Reply:
Removed, as I understand we don't need to have this test as we don't have prepared value and as the root is `[32]byte{}`